src/Meal.js

```diff
@@ -0,0 +1,8 @@
+class Meal {
```

Comment:
I am wondering if it is better to use a class or if it is better to use querySelectAll and insert HTML through JS? I think using a class means updating the Data Model, whereas the other option is updating the DOM?